remi/gui.py

```diff
@@ -519,338 +533,600 @@ class Widget(Tag, EventSource):
+    @editor_attribute_decorator("Font", '''Style''', 'DropDown', {'possible_values': (
+            'normal', 'bold', 'bolder', 'lighter', '100', '200', '300', '400', '500', '600', '700', '800', '900',
+            'inherit')})
+    def css_font_weight(self):
+        return self.style.get('font-weight', None)
+
     @css_font_weight.setter
-    def css_font_weight(self, value): self.style['font-weight'] = str(value)
+    def css_font_weight(self, value):
+        self.style['font-weight'] = str(value)
+
     @css_font_weight.deleter
-    def css_font_weight(self): del self.style['font-weight']
+    def css_font_weight(self):
+        del self.style['font-weight']
 
     @property
-    @editor_attribute_decorator("Font",'''Specifies how white-space inside an element is handled''', 'DropDown', {'possible_values': ('normal', 'nowrap', 'pre', 'pre-line', 'pre-wrap', 'initial', 'inherit')})
-    def css_white_space(self): return self.style.get('white-space', None)
+    @editor_attribute_decorator("Font", '''Specifies how white-space inside an element is handled''', 'DropDown', {
+        'possible_values': ('normal', 'nowrap', 'pre', 'pre-line', 'pre-wrap', 'initial', 'inherit')})
+    def css_white_space(self):
+        return self.style.get('white-space', None)
+
     @css_white_space.setter
-    def css_white_space(self, value): self.style['white-space'] = str(value)
+    def css_white_space(self, value):
+        self.style['white-space'] = str(value)
+
     @css_white_space.deleter
-    def css_white_space(self): del self.style['white-space']
+    def css_white_space(self):
+        del self.style['white-space']
 
     @property
-    @editor_attribute_decorator("Font",'''Increases or decreases the space between characters in a text.''', 'css_size', {})
-    def css_letter_spacing(self): return self.style.get('letter-spacing', None)
+    @editor_attribute_decorator("Font", '''Increases or decreases the space between characters in a text.''',
+                                'css_size', {})
+    def css_letter_spacing(self):
+        return self.style.get('letter-spacing', None)
+
     @css_letter_spacing.setter
-    def css_letter_spacing(self, value): self.style['letter-spacing'] = str(value)
+    def css_letter_spacing(self, value):
+        self.style['letter-spacing'] = str(value)
+
     @css_letter_spacing.deleter
-    def css_letter_spacing(self): del self.style['letter-spacing']
+    def css_letter_spacing(self):
+        del self.style['letter-spacing']
 
     @property
-    @editor_attribute_decorator("Layout",'''The flex-direction property specifies the direction of the flexible items. Note: If the element is not a flexible item, the flex-direction property has no effect.''', 'DropDown', {'possible_values': ('row', 'row-reverse', 'column', 'column-reverse', 'initial', 'inherit')})
-    def css_flex_direction(self): return self.style.get('flex-direction', None)
+    @editor_attribute_decorator("Layout",
```

Comment:
I know that a better indentation is fundamental for code reading, but here (as in all other indetation fixes on @properties) the indentation makes the code too long, preventing good readability. Here the code is verbose (almost all properties are identical) and so I prefer to make this code compact.